tools/config/project.config.ts

```diff
@@ -4,6 +4,21 @@ import {SeedConfig, normalizeDependencies} from './seed.config';
 export class ProjectConfig extends SeedConfig {
   PROJECT_TASKS_DIR = join(process.cwd(), this.TOOLS_DIR, 'tasks', 'project');
 
+  APP_TITLE = 'My Angular2 App';
+
+  // Autoprefixer configuration.
+  BROWSER_LIST = [
+    'ie >= 10',
+    'ie_mob >= 10',
+    'ff >= 30',
+    'chrome >= 34',
+    'safari >= 7',
+    'opera >= 23',
+    'ios >= 7',
+    'android >= 4.4',
+    'bb >= 10'
+  ];
+
   constructor() {
     super();
     // this.APP_TITLE = 'Put name of your app here';
```

Comment:
Think you're correct with the consistency with the other conventions of the seed.

Other option could be a setter method for app_title which you can call from the project.config.ts constructor and other settings which are usually set in the `project.config.ts` but that sounds/feels messy as well.

What are the most overridden options besides APP_TITLE (and probably BROWSER_LIST)?